### ISSUE TITLE
Improve multiple file selection

### ISSUE THREAD

**Issue by danicatalan** [huridocs/uwazi#829]
After #806 is implemented and tested, we need to think about some particular issues:

- [ ] How to do multiple select on mobile.
- [ ] How to teach users the cmd + click function.

**Comment by danicatalan** [comment]
Currently multiselect is used to edit multiple files. We're focused on develop UX for admins on desktop, so it doesn't make sense spend efforts on solve multiselect on mobile.

For the multiselect help text, I'm closing this in favor of #909.
